spec/requests/media_auth_request_spec.rb

```diff
@@ -2,17 +2,11 @@
 
 RSpec.describe "Authentication for Media requests", type: :request do
 
-  let(:allowed_loc) { 'ip.address1' }
   let(:user_no_loc_no_webauth) { User.new }
-  let(:user_no_loc_webauth) { User.new(webauth_user: true) }
```

Comment:
I believe that this behavior is tested (multiple times) between the ability tests (via user spec), the controller tests, the request tests, and the unit tests for the `MediaAuthenticationJSON`. As opposed to removing this test entirely, I thought it was useful for indicating that the JSON was being served up correctly under the broad use cases (and the individual use cases are tested in the other places).

If we retained all of the scenarios here, it felt like we were unnecessarily duplicating assertions already being made.